app/views/layouts/_header.html.erb

```diff
@@ -40,7 +40,7 @@
 <div class="logo">
     <a href="/">
         <div class="logo__image">
-            <img src="<%= asset_pack_path 'media/images/getintoteachinglogo.svg' %>" />
+            <img src="<%= asset_pack_path 'media/images/getintoteachinglogo.svg'%>" alt="Get into teaching" />
```

Comment:
You can probably have an empty alt here if it's a logo as it doesn't provide context